python/pylibcugraph/pylibcugraph/graphs.pyx

```diff
@@ -146,14 +141,14 @@ cdef class SGGraph(_GPUGraph):
                   GraphProperties graph_properties,
                   src_or_offset_array,
                   dst_or_index_array,
-                  vertices_array=None,
                   weight_array=None,
                   store_transposed=False,
                   renumber=False,
                   do_expensive_check=False,
                   edge_id_array=None,
                   edge_type_array=None,
                   input_array_format="COO",
+                  vertices_array=None,
```

Comment:
This (and also the updated MGGraph ctor) still does not have all new args at the end, but maybe since only `drop_self_loops` and `drop_multi_edges` are affected, it's not that likely to break users using only positional params?  I'll approve it but I'll also label this PR as a breaking change.